### ISSUE TITLE
Universal Forwarder

### ISSUE THREAD

**Issue by lukasmrtvy** [seunomosowon/TA-connectivity#2]
Is possible to modify UF to be able to run TA-connectivity addon?

Running:
```
apt update && apt install -y python
ln -s /usr/bin/python /opt/splunk/bin/python
ln -s /usr/lib/python2.7 /opt/splunk/bin/python2.7
```
Still there are some issues with splunk pyhon dependencies...

> python: can't open file '/opt/splunk/lib/python2.7/site-packages/splunk/clilib/cli.py': [Errno 2] No such file or directory

Difference between Heavy Splunk and UFW is like 500mb in size...

**Comment by seunomosowon** [comment]
Hi Lukas,

I'll see what I can do later. There's definitely a way to use python
externally.

Cheers

**Comment by lukasmrtvy** [comment]
Hello,
( site-packages/splunk* is not splunk-sdk. )
It cannot be downloaded, its embedded in splunk enterprise.
So only Splunk or Splunk Heavy Forwarder can run this addon.

**Comment by seunomosowon** [comment]
Yes. It needs to be run on a full Splunk enterprise instance or HF as it
is, but I could add a script to this that can be called externally.
See this the read me here https://splunkbase.splunk.com/app/1473/#/details
